.github/workflows/ci.yaml

```diff
@@ -327,6 +327,33 @@ jobs:
           name: gradle_jdbc_sqlserver_integration_test_reports
           path: core/build/reports/tests/integrationTestJdbc
 
+  integration-test-for-jdbc-sqlite:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up JDK 8
+        uses: actions/setup-java@v3
+        with:
+          java-version: '8'
+          distribution: 'temurin'
+
+      - name: Set up SQLite3
+        run: sudo apt-get install -y sqlite3
+
+      - name: Setup and execute Gradle 'integrationTestJdbc' task
+        uses: gradle/gradle-build-action@v2
+        with:
+          arguments: integrationTestJdbc -Dscalardb.jdbc.url=jdbc:sqlite:integration.sqlite3?busy_timeout=50000
```

Comment:
Since SQLite3 poorly handles parallel connections (see https://fanblogs.jp/scripts/archive/90/0), parallel integration tests like `put_withPutIfWithMultipleConditions_shouldPutProperly()` fail without the [`busy_timeout` parameter](https://www.sqlite.org/c3ref/busy_timeout.html).

Since ScalarDB users would frequently observe `CrudConflictException` via SQLITE_BUSY errors without `busy_timeout` option, I would like to recommend users to add the parameter in https://github.com/scalar-labs/scalardb/blob/master/docs/getting-started-with-scalardb-on-jdbc.md (in another PR).

Reply:
Did it in https://github.com/scalar-labs/scalardb/pull/838